### ISSUE TITLE
Volvo EX30 single motor 49kWh seems to have a broken charging curve

### ISSUE THREAD

**Issue by jonasman** [KilowattApp/open-ev-data#2]
```
"charging_curve": [
                    {
                        "percentage": 50,
                        "power": 0.0
                    },
                    {
                        "percentage": 60,
                        "power": 10.0
                    },
                    {
                        "percentage": 79,
                        "power": 20.0
                    },
                    {
                        "percentage": 98,
                        "power": 30.0
                    },
                    {
                        "percentage": 98,
                        "power": 30.0
                    },
                    {
                        "percentage": 138,
                        "power": 50.0
                    },
                    {
                        "percentage": 148,
                        "power": 60.0
                    },
                    {
                        "percentage": 80,
                        "power": 70.0
                    },
                    {
                        "percentage": 71,
                        "power": 80.0
                    },
                    {
                        "percentage": 63,
                        "power": 100.0
                    }
                ],
```

**Comment by jonasman** [comment]
It seems that power and percentage are reverted.

**Comment by tijs** [comment]
values were reversed indeed, good catch. fixed in #4
